recipes/seroba/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "seroba" %}
+{% set version = "1.0.0" %}
+{% set md5sum = "fcd6802070aa90d3f4810bb72a23cdce" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/sanger-pathogens/{{ name }}/archive/v{{ version }}.tar.gz
+  md5: {{ md5sum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - mummer
+    - bowtie2
+    - cd-hit
+    - ariba >=2.9.1
+    - pymummer >=0.10.2
+    - biopython >=1.68
+    - pyyaml >=3.12
+    - pyfastaq >=3.14.0
+    - wget
+  run:
+    - mummer
+    - bowtie2
+    - cd-hit
+    - ariba >=2.9.1
+    - pymummer >=0.10.2
+    - biopython >=1.68
+    - pyyaml >=3.12
+    - pyqt ==4.11.4
+    - pyfastaq >=3.14.0
+    - glib
+    - xorg-libxext 
+    - xorg-libsm
+    - xorg-libxrender
+test:
+  commands:
+    - seroba -h
+
+about:
+  home: https://github.com/sanger-pathogens/seroba
+  license: GPL3.0
+  license_family: GPL
+  summary: 'SeroBA is a k-mer based Pipeline to identify the Serotype from Illumina NGS reads for given references.'
+
+description: |
+      SeroBA is a k-mer based Pipeline to identify the Serotype from Illumina NGS reads for given references. 
+      You can use SeroBA to download references from (https://github.com/phe-bioinformatics/PneumoCaT) 
+      to do identify the capsular type of Streptococcus pneumoniae.
```

Comment:
Please append
```yaml
extras:
  skip-lints:
    - uses_setuptools  # uses pkg_resources
```

Reply:
After adding this to the recipe, it still fails linting for some reason. Passes on local `./simulate-travis.py`

Reply:
Very sorry about my typo!

Reply:
Sorry, which is the typo? :laughing:

Reply:
The surplus `s` in `extras` that you had to track down :X :wink: